server/route/api.js

```diff
@@ -0,0 +1,58 @@
+const express = require('express')
+const router = express.Router()
+const axios = require('axios');
+const City = require('./DBModels/City.js')
+
+router.get('/',function(req,res){
+    console.log("ohh hi new visitor")
+    res.send("ok working")
+})
+
+router.get(`/city/:cityName`,async function(req,res){
+    const cityName = req.params.cityName
+    axios.get(`http://api.openweathermap.org/data/2.5/weather?q=${cityName}&appid=561ae52a3ff46bd22a23af0e1667cbf1`)
```

Comment:
make `.env` file for you api keys. It is important to protect them and it does not look good if you post to github your key